openy_gated_content.install

```diff
@@ -243,3 +243,18 @@ function openy_gated_content_update_8012() {
       ],
     ]));
 }
+
+/**
+ * Added empty block components settings.
+ */
+function openy_gated_content_update_8013() {
+  $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
+  $config = drupal_get_path('module', 'openy_gated_content');
+  $config .= '/config/install/';
+  $config .= 'openy_gated_content.settings.yml';
+  $config_importer->update(
+    $config,
+    'openy_gated_content.settings',
+    'components'
```

Comment:
This hook_update could update someone's already entered data.
For example, if someone disabled videos component at the home page, it will return back.
Let me find another example for you.

Reply:
https://github.com/fivejars/openy_gated_content/blob/1.3/modules/openy_gc_auth/modules/openy_gc_auth_daxko_barcode/openy_gc_auth_daxko_barcode.install
This could be an example.

Reply:
@anpolimus, I have made the changes to update only empty block text configurations. Please do review